Tests/FlatMapLatestTests.swift

```diff
@@ -9,7 +9,8 @@
 #if !os(watchOS)
 import XCTest
 import Combine
-import CombineExt
+@testable import CombineExt
```

Comment:
Why `@testable` is needed ?

Reply:
Great catch, this is a leftover change left from an intermediary commit that I later squashed.